packages/experiments/src/components/Microfeedback/Microfeedback.styles.ts

```diff
@@ -0,0 +1,7 @@
+import { IMicrofeedbackStyleProps, IMicrofeedbackStyles } from './Microfeedback.types';
+
+export const getStyles = (props: IMicrofeedbackStyleProps): IMicrofeedbackStyles => {
+  return {
+    root: []
```

Comment:
Will this be filled in at all? If not, it could be removed.

Reply:
Fixed.